src/non_zero.rs

```diff
@@ -233,7 +233,7 @@ where
     T: Random + Zero,
 {
     /// Generate a random `NonZero<T>`.
-    fn random(mut rng: &mut impl CryptoRngCore) -> Self {
+    fn random(mut rng: &mut impl RngCore) -> Self {
```

Comment:
It may be useful to move aspects of the internal comment to the doc comment now that a CSPRNG is no longer guaranteed.

Reply:
I removed this doc comment and allowed the `Random::random()` one take over.